### PR TITLE
Changes graphql endpoint according to context directive

### PR DESCRIPTION
**Pull request by tlgimenes**
Sends graphql query to endpoint /_v/v{version}/${scope}/graphql with defaults of version = 1 and scope = public

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "apollo-cache-inmemory": "^1.1.5",
     "apollo-client": "^2.2.0",
+    "apollo-link-batch-http": "^1.2.1",
```
Comment (firstdoit) [inline react/package.json]: Is this necessary?

File: react/utils/client/index.ts [hunk 2]
```diff
@@ -24,29 +25,28 @@ export const getState = (runtime: RenderRuntime) => {
 }
 
 export const getClient = (runtime: RenderRuntime) => {
-  const {graphQlUri, account, workspace} = runtime
+  const {account, workspace} = runtime
```
Comment (firstdoit) [inline react/utils/client/index.ts]: We will need the `graphQlUri` in other parts of the runtime, like fetching messages. @brunoabreu @tlgimenes how do we go around that?

File: react/package.json (hunk 1 above)
Comment (tlgimenes) [reply react/package.json]: Sorry, interferences from other features, changed now

File: react/utils/client/index.ts (hunk 2 above)
Comment (tlgimenes) [reply react/utils/client/index.ts]: I have no idea. I think you and @brunoabreu  can make a patch to this branch solving this problem

File: react/utils/client/generateHash.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  Argument(node: ArgumentNode) {
+    if (node.name.value === 'hash') {
+      asset.hash = node.value.value
+    }
+  }
+})
+
+export const generateHash = (query: DocumentNode) => {
+  if (query.documentId) {
+    return query.documentId
```
Comment (brunoabreu) [inline react/utils/client/generateHash.ts]: What is this `documentId`? Is it the same as `hash`?

Comment (tlgimenes) [reply react/utils/client/generateHash.ts]: Yes, it should be there for legacy compatibility

**Pull request merged**